TestCase/Managers/InputManager.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Runtime.Intrinsics.X86;
+
+namespace TestCase.Managers
+{
+    public class InputManager
+    {
+        public int GetInput()
+        {
+            Console.Write("\nEnter number between 1 and 3: ");
+
+            if (int.TryParse(Console.ReadLine(), out int num))
+            {
+                if (num >= 1 && num <= 3)
+                {
+                    return num;
+                }
+                else
```

Comment:
The else here is redundant. If you leave it itll go the the code at the end of the method, the exact same 👍